Softeq.XToolkit.Common.iOS/Softeq.XToolkit.Common.iOS.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0-ios12.0</TargetFramework>
+    <TargetFramework>net8.0-ios12.0</TargetFramework>
```

Comment:
Can we use iOS 17?

Reply:
The same as for `Softeq.XToolkit.PushNotifications.iOS.csproj`
ios17 is supported